go.mod

```diff
@@ -1,20 +1,24 @@
 module github.com/infobloxopen/protoc-gen-gorm
 
 require (
-	github.com/golang-jwt/jwt/v4 v4.4.1
+	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/golang/protobuf v1.5.2
 	github.com/google/go-cmp v0.5.8
-	github.com/grpc-ecosystem/go-grpc-middleware v1.2.2
-	github.com/infobloxopen/atlas-app-toolkit v1.2.2-0.20220721202134-28480c4993fc
+	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
+	github.com/infobloxopen/atlas-app-toolkit v1.4.0
+	github.com/infobloxopen/atlas-app-toolkit/v2 v2.2.1-0.20240313220428-5449c0c2a27f
```

Comment:
If this was from `go get github.com/infobloxopen/atlas-app-toolkit/v2@gorm_v2`, you should probably make a note for future reference